### PR TITLE
Testing iteration1

### PR DESCRIPTION
**Pull request by TimB-87**
Intake, Shooter, Arm Rotation. They all work! They're all tuned!

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -19,16 +19,19 @@
...
 import frc.robot.subsystems.Intake;
 import frc.robot.subsystems.Led;
 import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.ArmRotate.ArmPresets;
+import frc.robot.subsystems.Shooter.Speeds;
 
 public class Robot extends CommandRobot {
 
-    private RobotMap map = getRobotMap(RobotMap.class, new RobotMap());
+    private RobotMap map = new Henry();
```
Comment (msoucy) [inline src/main/java/frc/robot/Robot.java]: Nope. Use the reflection method.

File: src/main/java/com/chopshop166/chopshoplib/sensors/CSDutyCycleEncoderLocal.java [hunk 2]
```diff
@@ -44,7 +44,11 @@ public double getAbsolutePosition() {
         if (distance == -this.positionOffset) {
             return 0;
         }
+        if (distance < -10) {
```
Comment (msoucy) [inline src/main/java/com/chopshop166/chopshoplib/sensors/CSDutyCycleEncoderLocal.java:47]: Why was this added?

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -86,13 +89,25 @@ public void configureButtonBindings() {
...
+        // copilotController.povRight().onTrue(shooter.setSpeed(Speeds.HALF_SPEED));
+        copilotController.povRight().whileTrue(armRotate.moveTo(ArmPresets.INTAKE));
+        copilotController.povDown().onTrue(shooter.setSpeed(Speeds.SLOW_SPEED));
+        copilotController.povLeft().onTrue(shooter.setSpeed(Speeds.THREE_QUARTER_SPEED));
+        copilotController.start().onTrue(shooter.setSpeed(Speeds.OFF));
+        copilotController.y().onTrue(intake.feedShooter());
+        copilotController.x().whileTrue(intake.spinOut());
+        // copilotController.povUp().whileTrue(commandSequences.moveToAmp());
+        // copilotController.povDown().whileTrue(commandSequences.moveToIntake());
+        // copilotController.povRight().whileTrue(commandSequences.moveToSpeaker());
```
Comment (msoucy) [inline src/main/java/frc/robot/Robot.java]: I don't love the commented out code. Do we still need this?

**Review by msoucy — CHANGES_REQUESTED** [review]

**Comment by msoucy** [comment]
Needs a rebase

File: src/main/java/com/chopshop166/chopshoplib/sensors/CSDutyCycleEncoderLocal.java (hunk 2 above)
Comment (Liammurray19) [reply src/main/java/com/chopshop166/chopshoplib/sensors/CSDutyCycleEncoderLocal.java:47]: The arm broke and every once in a while it went -90 so we added 180. It also needs to be -10 because sometimes it goes to -2 or something and we don't want it going to 180 when at basically 0.

**Review by msoucy — APPROVED** [review]

**Review by bot190 — APPROVED** [review]
